media-video/pipewire/metadata.xml

```diff
@@ -10,11 +10,20 @@
 		<name>Andreas Sturmlechner</name>
 	</maintainer>
 	<upstream>
-		<remote-id type="github">PipeWire/pipewire</remote-id>
```

Comment:
While the <remote-id type="gitlab"> does exist, it almost certainly is meant for referencing to gitlab.com hosted projects and not gitlab.freedesktop.org or other hosts using GitLab software. One would imagine that freedesktop should have its own remote-id type but after looking thrice and in various places, I still could not find such a thing.

Reply:
I have confirmed with Gentoo devs that that is, indeed, the case. Closing this.